test/controllers/home_controller_test.rb

```diff
@@ -1,9 +0,0 @@
-require 'test_helper'
-
-class HomeControllerTest < ActionDispatch::IntegrationTest
```

Comment:
why remove this test???

Reply:
Its a placeholder test from the test folder. We are replacing `test` with `spec`